### PR TITLE
TravisCI: add support for ppc64le

### PR DESCRIPTION
**Pull request by sanjay-cpu**
Hi,
I have added support for ppc64le build on travis-ci in the branch . The travis-ci build log can be tracked on the link :https://travis-ci.com/github/sanjay-cpu/libvncserver/builds/182311847 . I believe it is ready for the final review and merge.
Please have a look on it and if everything looks fine for you then please approve it for merge.

Thanks !!

### CODE REVIEW

**Comment by sanjay-cpu** [comment]
Hi, can someone please look into this PR? Thanks!!

**Comment by bk138** [comment]
It's on my agenda, just lacking time :-)

File: .travis.yml [hunk 1]
```diff
@@ -16,9 +20,13 @@ jobs:
   exclude:
     - os: osx
       env: CMAKE_OPTIONS="-DCMAKE_TOOLCHAIN_FILE=../cmake/Toolchain-cross-mingw32-linux.cmake" # don't test MinGW from OSX
+    - os: linux
+      env: CMAKE_OPTIONS="-DCMAKE_TOOLCHAIN_FILE=../cmake/Toolchain-cross-mingw32-linux.cmake"
```
Comment (bk138) [inline .travis.yml:24]: A clarifying comment as in line 22 would be nice here.

File: .travis.yml [hunk 2]
```diff
@@ -16,9 +20,13 @@ jobs:
...
+      arch: ppc64le
 
 before_install:
-  - if [ "$TRAVIS_OS_NAME" = "linux" ]; then sudo apt-get update; sudo apt-get --no-install-suggests --no-install-recommends install libsdl2-dev liblzo2-dev gnutls-dev libgcrypt-dev wine; fi
+  - if [ "$TRAVIS_OS_NAME" = "linux" -a "$TRAVIS_ARCH" = "amd64" ]; then sudo apt-get install wine ; fi
```
Comment (bk138) [inline .travis.yml:28]: please nest the tests so there is only a single test for "linux". also:
* apt-get update should be called in any case
* apt-get install should be called with the given --options in any case

**Review by bk138 — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (sanjay-cpu) [reply .travis.yml]: Thanks @bk138 , I have added the comment.

File: .travis.yml (hunk 2 above)
Comment (sanjay-cpu) [reply .travis.yml]: Hi @bk138 , I have nested the tests in a single test for linux as asked.

Thanks !!

**Comment by sanjay-cpu** [comment]
It looks like all the issues are resolved and it is ready for the final review and merge.
@bk138  Please have a look on it.

Thanks !!

**Comment by sanjay-cpu** [comment]
@bk138 Thanks for accepting the PR.

**Pull request merged**